apps/emqx_utils/src/emqx_variform_bif.erl

```diff
@@ -199,20 +199,24 @@ regex_match(Str, RE) ->
 regex_replace(SrcStr, RE, RepStr) ->
     re:replace(SrcStr, RE, RepStr, [global, {return, binary}]).
 
-%% @doc Searches the string Str for patterns specified by Regexp.
+%% @doc Non-global search for specified by regular expression patter in the given string.
```

Comment:
Non-global search for specified by regular expression pattern in the given string.